src/components/atoms/DateTimePicker/Component.tsx

```diff
@@ -213,42 +213,42 @@ const styles = StyleSheet.create({
   },
   iconContainer: {
     position: 'absolute',
-    right: defaultTheme?.spacing?.SPACING_10,
+    right: 10,
```

Comment:
using px directly because of not using scalable function on mobile editor

Reply:
hmm, i think we should think about it later.

Reply:
can you create a issue about it @sieuhuflit